.travis.yml

```diff
@@ -11,6 +11,7 @@ install: bundle install --without=development
 
 rvm:
   - rbx-2
+  - rbx-3.37
```

Comment:
This ought to be `rbx` w/o a version, and definitely not `3.37` -- about to push out a revised travis config in general though